src/extension.ts

```diff
@@ -558,6 +558,15 @@ async function getSopsGeneralOptions() {
 		sopsGeneralEnvVars[AGE_KEY_FILE_ENV_VAR_NAME] = ageKeyFile;
 	}
 
+	// Needed to work with sops via aqua (https://github.com/aquaproj/aqua).
+	// Error related to this issue https://github.com/getsops/sops/issues/884 is thrown otherwise.
+	sopsGeneralArgs.push('--config', '/dev/null');
+	// Moreover sops via aqua need to be executed from directory inside repository where config file aqua.yaml exists
+	// Ensuring that sops is executed from directory where .sops.yaml is found permit to use sops via aqua in case of a repository
+	// If .sops.yaml is not found, sops is executed from user home directory like before
+	let sopsConfigUri = await findSopsConfigRecursive(uri.with({ path: path.dirname(uri.path) }));
+	spawnOptions.cwd = sopsConfigUri ? path.dirname(sopsConfigUri.path) : spawnOptions.cwd;
```

Comment:
You can use the if statement here to assign a new value instead of ternary. It's a little bit more readable, and you reduce the operation of reassigning the same value here.

Reply:
Using if statement instead of ternary in this commit [1c64602](https://github.com/signageos/vscode-sops/pull/73/commits/1c64602fc4a21a9c9863a03eb10500d6e2f2d294)